src/main/java/com/teragrep/jpr_01/JavaPcre.java

```diff
@@ -169,10 +169,10 @@ public static class ByValue extends RegexStruct implements Structure.ByValue {}
     LibJavaPcre.OptionsStruct compile_options;
     LibJavaPcre.ExtraOptionsStruct extra_options;
     LibJavaPcre.MatchOptionsStruct match_options;
-    Map<String, Integer> name_table;
-    Map<Integer, String> match_table;
+    public Map<String, Integer> name_table;
```

Comment:
instead of direct member field access, change to getter pattern

Reply:
please also make all member fields private, if not absolutely required to be package-private